Resources/translations/messages.en.xlf

```diff
@@ -7,8 +7,8 @@
     </header>
     <body>
       <trans-unit id="07ec12f4689ee90ade282338b25b99b9677b0d08" resname="form.validation_error">
-        <source>Invalid form data</source>
-        <target>Invalid form data</target>
+        <source>Form did not validate. Please review errors below.</source>
+        <target>Form did not validate. Please review errors below.</target>
```

Comment:
I've changes this translation because many behat scenarios assumed that validation error will look like this ;-)